packages/ui-components/src/CopyButton.tsx

```diff
@@ -28,30 +24,24 @@ const StyledCopyButton = styled.button`
   }
 `;
 
-export class CopyButton extends React.PureComponent<Props, State> {
-  state: State = {
-    copied: false
-  };
+export function CopyButton (props: CopyButtonProps) {
+  const { value } = props;
+  const [copied, setCopied] = useState(false);
```

Comment:
it looks like you're specifying the state types in other components, so even though it's obvious here what the types are I think it'd be worth doing so here as well for consistency. `useState<boolean>(false)`

Reply:
Tbh I'd prefer not to put explicitly. TS type inferring works pretty well, and someone seeing `useState(false)` knows immediately it's a boolean.

The reason why I put in some cases is that TS cannot infer the type, and needs a hint.